src/App.tsx

```diff
@@ -19,16 +44,30 @@ export const App: React.FC = () => {
             <div className="block">
               <TodoFilter />
             </div>
-
             <div className="block">
-              <Loader />
-              <TodoList />
+              {
+                isLoading && <Loader />
+              }
+
+              {
+                !hasErrorHappened && !isLoading && <TodoList />
+              }
+
+              {
+                hasErrorHappened && !isLoading && (
+                  <p className="notification is-danger">
+                    There are no todos matching current filter criteria
+                  </p>
```

Comment:
You should show this message when a user tries to find a todo with wrong input text, not when you get an error during loading todos. In this case when error is happened you can show error notification like 'Unable to load todos'.